docker-compose.yml

```
@@ -23,7 +23,7 @@ services:
       - dev
 
   keycloak: &keycloak
-    image: ${KEYCLOAK_VERSION-keycloak}
+    image: jboss/keycloak:${KEYCLOAK_VERSION-11.0.0}
```

```suggestion
    image: ${KEYCLOAK_IMAGE-keycloak}
```

The current version is my fault, I've used the wrong env var. `KEYCLOAK_IMAGE` can be set to any repo, or we can use the locally named `keycloak` (as default) to use whatever has been built previously.

---

@GermanDZ I've never had that variable set anywhere... Why would we want to change the name of the image? I think the version is the important part right?

I could do something like that maybe:

```suggestion
    image: ${KEYCLOAK_IMAGE}:${KEYCLOAK_VERSION}
```

---

I think we will mostly use an image built by ourselves (or infra), named `keycloak`. But sometimes, for upgrades, hacking, testing you can try to use another image (and older version number, from another registry, etc).

Right now I'm not using the env var, I'm always using `keycloak`.

---

Got you yeah I forgot that it was based on the docker image!